inc/task.class.php

```diff
@@ -96,27 +118,28 @@ public static function checkReservations($task) {
                         WHERE `reservations_id`='" . $reservation->fields["id"] . "'";
                $DB->query($query) or die("error on 'update' into checkReservations conflict : " . $DB->error());
             } else {
-               $task->log("conflit avec la reservation " . $conflict_reservation->fields['id'] . " du materiel " . $item->fields['name'] . " par " . $formatName . " (du " . date("d-m-Y \à H:i:s", strtotime($conflict['begin'])) . " au " . date("d-m-Y \à H:i:s", strtotime($conflict['end']).")"));
-               NotificationEvent::raiseEvent('plugin_reservation_conflict', $conflict_reservation);
+               $task->log("conflit for reservation " . $conflict_reservation->fields['id'] . " ontiem " . $item->fields['name'] . " used by " . $formatName . " (from " . date("d-m-Y \à H:i:s", strtotime($conflict['begin'])) . " to " . date("d-m-Y \à H:i:s", strtotime($conflict['end']).")"));
+               NotificationEvent::raiseEvent('plugin_reservation_conflict_new_user', $conflict_reservation, ['other_user_id' => $res['users_id']]);
```

Comment:
I think it's better to switch names : conflict_new_user for the next user (user of the next reservation), and conflict_previous_user for the current reservation user, no ?!